XU-TechnicalPaper.md

```diff
@@ -153,11 +153,15 @@ Makers are earning XUC, takers are paying for filling orders. Makers set a price
 
 4. Punish malicious behavior
 
-Market makers are required to stake a certain amount of XUC in a smart contract in order to be able to act as a market maker. This will be used to execute punishments, for example if a makers XUD creates HTLCs for a payment, but never releases the preimage to execute them and thus locks up funds of other exchanges.
+All XUD's are required to stake a certain amount of XUC in a Ethereum smart contract in order to be able to act as a maker or taker. For the PoC, the staking minimum is a flat 1000 XUC, in a later phase the amount of staked XUC will determine the overall volume a XUD can trade (denominated in XUC, fed by several price feeds). This staked XUC will be taken away as punishment in case of malicious behavior. It's not part of the PoC. 
+
+The XUD setup procedure is as follows: the XUC hot wallet is funded with 1000 XUC, which then deposits these into a smart contract, which is controlled by a set of watchtowers (X-of-X multisig). Watchtowers maintain a list which `XUD_IDs` staked XUC and XUDs can query a set of watchtowers for specific `XUD_IDs` to verify they indeed did stake the required XUC. 
+
+If a XUD in a maker role behaves maliciously, for example creates HTLCs for a payment, but never releases the preimage to execute them and thus locks up funds of taker(s), the payment channel is force-closed by the taker after X amount of blocks. Block times differ from chain to chain, the default will be set to block times corresponding to about one hour. Since it is the taker broadcasting the force-close transaction, the maker will be given the one hour grace period to veto this punishment via a cryptographic proof that the swap completed successfully - the revoke transaction. Watchtowers maintain full nodes of multiple chains and watch for confirmed (not revoked) force-closed transactions. If one is found, the staked XUC of the maker are transferred to another contract which burns them, if a successfully revoked transaction is found, the staked XUC of the taker are burned.
```

Comment:
I don't follow here. If the taker broadcasts an old state of the channel, they lose all their funds on the force-close commitment transactions when the penalty transaction is broadcast. But a force close transaction that doesn't get penalized isn't proof that the other party misbehaved. There's also other forms of misbehavior, namely spamming fake orders.

Reply:
New commit coming up.